File: generated-sources/scalaz/mojang-authentication/src/main/scala/com/github/asyncmc/mojang/authentication/scalaz/model/AuthenticationRequest.scala
package com.github.asyncmc.mojang.authentication.scalaz.model

import argonaut._
import argonaut.EncodeJson._
import argonaut.DecodeJson._

import org.http4s.{EntityDecoder, EntityEncoder}
import org.http4s.argonaut._
import org.joda.time.DateTime
import AuthenticationRequest._

case class AuthenticationRequest (
  /* The Mojang account e-mail or username. Never store it. */
  username: String,
/* The Mojang account password, never store it. */
  password: String)

object AuthenticationRequest {
  import DateTimeCodecs._

  implicit val AuthenticationRequestCodecJson: CodecJson[AuthenticationRequest] = CodecJson.derive[AuthenticationRequest]
  implicit val AuthenticationRequestDecoder: EntityDecoder[AuthenticationRequest] = jsonOf[AuthenticationRequest]
  implicit val AuthenticationRequestEncoder: EntityEncoder[AuthenticationRequest] = jsonEncoderOf[AuthenticationRequest]
}

File: generated-sources/php-laravel/mojang-authentication/lib/com/github/asyncmc/mojang/authentication/php/laravel/model/AuthenticationRequest.php
<?php
/**
 * AuthenticationRequest
 */
namespace com.github.asyncmc.mojang.authentication.php.laravel.server\com.github.asyncmc.mojang.authentication.php.laravel.model;

/**
 * AuthenticationRequest
 */
class AuthenticationRequest {

    /** @var string $username The Mojang account e-mail or username. Never store it.*/
    private $username;

    /** @var string $password The Mojang account password, never store it.*/
    private $password;

}
